svgelements/svgelements.py

```diff
@@ -3737,18 +3737,18 @@ def _repr_shape(self, values):
         """
         Generic pieces of repr shape.
         """
-        if not self.transform.is_identity():
-            values.append("transform=%s" % repr(self.transform))
         if self.stroke is not None:
-            values.append("stroke='%s'" % self.stroke)
+            values.append("%s='%s'" % (SVG_ATTR_STROKE, self.stroke))
         if self.fill is not None:
-            values.append("fill='%s'" % self.fill)
+            values.append("%s='%s'" % (SVG_ATTR_FILL, self.fill))
         if self.stroke_width is not None and self.stroke_width != 1.0:
-            values.append("stroke_width='%s'" % str(self.stroke_width))
+            values.append("%s=%s" % (SVG_ATTR_STROKE_WIDTH, str(self.stroke_width)))
```

Comment:
SVG_ATTR_STROKE_WIDTH == "stroke-width" The value stroke-width="value" cannot possibly work because - is not allowed within a variable name since it's stroke minus width. This will a priori fail the eval(repr()) test.